flask_peewee/rest/__init__.py

```diff
@@ -770,6 +795,46 @@ def delete(self, obj):
         res = obj.delete_instance(recursive=self.delete_recursive)
         return self.response({'deleted': res})
 
+    def json_value(self, obj, field_name, path):
+        value = getattr(obj, field_name)
+        for key in path:
+            if not isinstance(value, dict) or key not in value:
+                value = None
+                break
+            value = value[key]
+        return self.response(value)
+
+    def json_edit(self, obj, field_name, path):
+        value = getattr(obj, field_name)
+        for key in path[:-1]:
+            if key not in value or not isinstance(value[key], dict):
+                value[key] = {}
+            value = value[key]
+
+        key = path[-1]
+        value[key] = self.read_request_data()
+        obj.save()
+        return self.response(value[key])
+
+    def json_delete(self, obj, field_name, path):
+        value = getattr(obj, field_name)
```

Comment:
Similar comment to above about needing to handle the case of obj.field_name being None.